# Generated by Django 3.0.6 on 2020-05-22 08:22

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        ('patches', '0001_initial'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='patchApproverRelationship',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('approver', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='approverz', to=settings.AUTH_USER_MODEL)),
                ('patch', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='patches.patch')),
            ],
        ),
        migrations.CreateModel(
            name='authorize_Exception',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('exception_id', models.IntegerField(null=True)),
                ('state', models.CharField(choices=[('Approved', 'Approved'), ('Rejected', 'Rejected'), ('Pending', 'Pending')], default='Pending', max_length=8)),
                ('comment', models.TextField(blank=True, default='Pending')),
                ('approver', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='approverAuthorize', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
